sdk/metric/aggregation/aggregation.go

```diff
@@ -164,3 +164,54 @@ func (h ExplicitBucketHistogram) Copy() Aggregation {
 		NoMinMax:   h.NoMinMax,
 	}
 }
+
+// ExponentialHistogram is an aggregation that summarizes a set of
+// measurements as an histogram with bucket widths that grow exponentially.
+type ExponentialHistogram struct {
+	// MaxSize is the maximum number of buckets to use for the histogram.
+	MaxSize int
+	// MaxScale is the maximum resolution scale to use for the histogram.
+	MaxScale int
```

Comment:
The [proto](https://github.com/open-telemetry/opentelemetry-proto/blob/81a296f9dba23e32d77f46d58c8ea4244a2157a6/opentelemetry/proto/metrics/v1/metrics.proto#L515) encodes this with an `int32`.

```suggestion
	// MaxScale is the maximum resolution scale to use for the histogram.
	//
	// MaxScale has a maximum value of 20. Using a value of 20 means the
	// maximum number of buckets that can fit within the range of a
	// signed 32-bit integer index could be used.
	//
	// MaxScale has a minimum value of -10. Using a value of -10 means only
	// two buckets will be use.
	MaxScale int32
```